README.rst

```diff
@@ -155,3 +155,122 @@ Using your normal views over a websocket connection
       ),
    })
 
+
+Creating a fully-functional custom Consumer
+-------------------------------------------
+
+This package offers Django Rest Framework capabilities via mixins. To utilize these mixins, one must inherit from the GenericAsyncAPIConsumer. 
+
+One may use the same exact querysets and serializer_classes utilized in their DRF Views, but must omit the DRF permissions. 
+
+Permissions are to be imported from djangochannelsrestframework, which provides the standard AllowAny and IsAuthenticated permissions.
+
+
+.. code-block:: python
+
+    from . import models
+    from . import serializers
+    from djangochannelsrestframework import permissions
+    from djangochannelsrestframework.generics import GenericAsyncAPIConsumer
+    from djangochannelsrestframework.mixins import (
+        ListModelMixin,
+        PatchModelMixin,
+        UpdateModelMixin,
+        CreateModelMixin,
+        DeleteModelMixin,
+    )
+
+    class LiveConsumer(ListModelMixin, GenericAsyncAPIConsumer):
+        queryset = models.Test.objects.all()
+        serializer_class = serializers.TestSerializer
+        permission_classes = (permissions.IsAuthenticated,)
+
+
+Because this class uses the ListModelMixin, one has access to the `list` action.
+
+One can access this action from the client with a payload, or from within a method:
+
+.. code-block:: python
+
+    # From Client payload:
+    {action: "list", "request_id": 42}
```

Comment:
maybe break thes into 2 clear sections one for calling from a client over the websocket conneciton the other from within the list.